packages/api-clients/open-api/purposeTemplateApi.yml

```
@@ -0,0 +1,1064 @@
+          schema:
+            type: string
+        - in: query
+          name: creatorIds
+          description: comma separated sequence of creator IDs
+          schema:
+            type: array
+            items:
+              type: string
+              format: uuid
+            default: []
+          explode: false
+        - in: query
+          name: states
+          description: filter by template state
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/PurposeTemplateState"
+            default: []
+          explode: false
+        - in: query
+          name: offset
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 0
+        - in: query
+          name: limit
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 50
+      responses:
+        "200":
+          description: Purpose Templates requested
+          headers:
+            X-Metadata-Version:
+              $ref: "#/components/headers/MetadataVersionHeader"
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PurposeTemplate"
+        "400":
+          description: Bad Request
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+    post:
+      tags:
+        - purposeTemplate
+      operationId: createPurposeTemplate
+      description: Creates the Purpose Template (Draft state)
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PurposeTemplateSeed"
+        required: true
+      responses:
+        "201":
+          description: Purpose Template created
+          headers:
+            X-Metadata-Version:
+              $ref: "#/components/headers/MetadataVersionHeader"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PurposeTemplate"
+        "400":
+          description: Bad Request
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+  /purposeTemplates/eservices:
+    get:
+      tags:
+        - purposeTemplate
+      operationId: getEServices
+      description: Retrieve associations between all purpose templates and all their linked eservices.
```

Minor: alignments (double check in other endpoints) 
- no dots at the end of descriptions usually
- entities/resources are usually capitalized

```suggestion
      description: Retrieve associations between all Purpose Templates and all their linked E-Services
```

---

Comment still unaddressed in different endpoints (there are still some occurencies of non-capitalized entities or trailing dots ([example](https://github.com/pagopa/interop-be-monorepo/pull/2199/files#diff-0f2c99cee2118f23ac980539179dcc2813318d0ba2667ab710408cf2dd9d2ff6R218))